File: app/src/main/java/com/example/android/popularmovies/model/Review.java
package com.example.android.popularmovies.model;



public class Review {
    private String mUsername;
    private String mReview;

    public Review() {
    }

    public Review(String mUsername, String mReview) {
        this.mUsername = mUsername;
        this.mReview = mReview;
    }

    public String getmUsername() {
        return mUsername;
    }

    public void setmUsername(String mUsername) {
        this.mUsername = mUsername;
    }

    public String getmReview() {
        return mReview;
    }

    public void setmReview(String mReview) {
        this.mReview = mReview;
    }
}

File: app/src/main/java/com/example/android/popularmovies/model/Review.kt
package com.example.android.popularmovies.model

import javax.annotation.Generated
import com.google.gson.annotations.SerializedName

@Generated("com.robohorse.robopojogenerator")
class Review {

  @SerializedName("id")
  var id: Int = 0

  @SerializedName("page")
  var page: Int = 0

  @SerializedName("total_pages")
  var totalPages: Int = 0

  @SerializedName("results")
  var results: List<ReviewItem>? = null

  @SerializedName("total_results")
  var totalResults: Int = 0

  override fun toString(): String {
    return "Review{id = '$id${'\''.toString()},page = '$page${'\''.toString()},total_pages = '$totalPages${'\''.toString()},results = '$results${'\''.toString()},total_results = '$totalResults${'\''.toString()}}"
  }
}